### PR TITLE
chore: fix README mention about PV and VG creation

### LINKED ISSUE #343: not a bug, just incredibly confusing doc.
The doc now says you shouldn't provision the lvm volume group etc.  Is that true for lvm-localpv?  
How do I tell it which disks it should touch!

### PR DESCRIPTION
**Pull request by dsharma-dc**
## Pull Request template

Please, go through these steps before you submit a PR.

**Why is this PR required? What issue does it fix?**:
Documentation

**What this PR does?**:

**Does this PR require any upgrade changes?**:

**If the changes in this PR are manually verified, list down the scenarios covered:**:

**Any additional information for your reviewer?** :
_Mention if this PR is part of any design or a continuation of previous PRs_

**Checklist:**
- [x] Fixes #343
- [x] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [ ] Has the change log section been updated?
- [ ] Commit has unit tests
- [ ] Commit has integration tests
- [ ] (Optional) Are upgrade changes included in this PR? If not, mention the issue/PR to track:
- [ ] (Optional) If documentation changes are required, which issue on https://github.com/openebs/openebs-docs is used to track them:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,9 +53,8 @@ The orignal v1.0 dev roadmap [is here ](https://github.com/orgs/openebs/projects
 <BR>
 
 <!-- > [!NOTE]
-> - Full LVM2 dynamic provisioning is now supported <BR>
-> - All Volume Groups (VG), Physical Volumes (PV) and Logical Volumes (LV) will be dynamically provisioned for you by OpenEBS <BR>
-> - There is need to manually provision any PV, VG or LV's  <BR>
+> - LocalPV-LVM will not provision the VG for the user <BR>
+> - The required Physical Volumes(PV) and Volume Group(VG) names will need to be created and present beforehand. <BR>
 > - Dynamic RAID provisioning is still in development. RAID LV's must be deployed via the manual pre configuration method <BR> -->
```
Comment (tiagolobocastro) [inline README.md:58]:
```suggestion
```

**Review by tiagolobocastro — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Abhinandan-Purkait) [reply README.md:58]: But isn't this true?

Comment (dsharma-dc) [reply README.md:58]: I think yes, manual intervention is what users need to do. So can keep this.

**Review by abhilashshetty04 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (tiagolobocastro) [reply README.md:58]: This is not true, RAID is not under development right?

Comment (Abhinandan-Purkait) [reply README.md:58]: The raid pr that we have, does that not cater? Yeah maybe we should not mention it right now?

Comment (dsharma-dc) [reply README.md:58]: The PR is there, needs refinement and addressing so effectively in development I say.

**Pull request merged**